week-1/exercises/decimal.js

```diff
@@ -8,3 +7,21 @@ const num = 56.5467;
 // Create a variable called roundedNum and assign to it an expression that evaluates to 57 ( num rounded to the nearest whole number )
 
 // Log your variables to the console to check your answers
+
+const wholeNumberPart = Math.floor(num);
+
+console.log(wholeNumberPart);
+
+// const decimalPart = num - wholeNumberPart;
+
+// const decimalPart = num % 1;
+
+// const decimalPart = ((num - wholeNumberPart) * 10000) / 10000;
+
+const decimalPart = (num - wholeNumberPart).toFixed(4);
```

Comment:
Nice work on identifying that .toFixed is needed

Again, feel free to remove lines of code that are not useful!

Reply:
Will do. Thanks again!